pom.xml

```diff
@@ -83,10 +83,10 @@ THE SOFTWARE.
     <properties>
         <revision>1.51</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.176.4</jenkins.version>
+        <jenkins.version>2.204.6</jenkins.version>
         <java.level>8</java.level>
         <jcasc.version>1.36</jcasc.version>
-        <aws-java-sdk.version>1.11.723</aws-java-sdk.version>
+        <aws-java-sdk.version>1.11.821</aws-java-sdk.version>
```

Comment:
After a second thought, I realize: this is going to force this plugin to require this release with new additions, which is good. 

But this was not strictly necessary for a Jenkins instance to be able to see the new types: one would just have to make sure to update to the latest `aws-java-sdk` Jenkins plugin (i.e. 1.11.821+) right?